cosmos/highbury_710.json

```diff
@@ -0,0 +1,75 @@
+{
+  "rpc": "https://gridiron.furya.io",
+  "rest": "https://api-mainnet.furya.io",
+  "chainId": "highbury_710-1",
+  "chainName": "Highbury",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/highbury_710/chain.png",
+  "nodeProvider": {
+      "name": "Blackfury Genesis Validator",
+      "email": "blackfuryan@skiff.com",
+      "website":"https://fury.black/"
+    },
+  "walletUrl": "https://wallet.keplr.app/chains/highbury",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/highbury",
```

Comment:
@sudopunk Could you remove this two(`walletUrl`, `walletUrlForStaking `) filed?
This is an invalid URL and is not needed now.

Reply:
Yes, removed.